sysutils/gptfdisk/Portfile

```diff
@@ -0,0 +1,39 @@
+# $Id:  $
+
+PortSystem              1.0
+
+name                    gptfdisk
```

Comment:
Both _gdisk_ and _gptfdisk_ seem to be in use:

 * https://repology.org/metapackage/gptfdisk/versions
 * https://repology.org/metapackage/gdisk/versions

Whatever is chosen, the other name should be in the (long_)description, so it can be found with a search.